Gateway/Request/RefundDataBuilder.php

```diff
@@ -241,6 +241,28 @@ protected function getOpenInvoiceData($payment)
             );
         }
 
+        // Adjustment positive
+        if ($creditMemo->getAdjustment() != 0) {
+            $description = "Adjustment";
+            if ($creditMemo->getAdjustmentPositive() > 0 && $creditMemo->getAdjustmentNegative() > 0) {
+                $description .= " - Positive | Negative";
+            } elseif ($creditMemo->getAdjustmentPositive() > 0) {
+                $description .= " - Positve";
+            } elseif ($creditMemo->getAdjustmentNegative() > 0) {
+                $description .=  " - Negative";
+            }
```

Comment:
Maybe this is cleaner and still works the same way
```suggestion
        if ($creditMemo->getAdjustment() != 0) {
            $positive = $creditMemo->getAdjustmentPositive() > 0 ? 'Positive' : '';
            $negative = $creditMemo->getAdjustmentNegative() > 0 ? 'Negative' : '';
            $description = "Adjustment - " . implode(' | ', array_filter([$positive, $negative]));
```